nuxeo-distribution/nuxeo-nxr-server/src/main/resources/templates/common-base/nxserver/config/stream-config.xml.nxftl

```diff
@@ -16,7 +16,8 @@
       <option name="kafkaConfig">default</option>
     </logConfig>
     <logConfig name="bulk" type="kafka">
-      <option name="kafkaConfig">default</option>
+      <option name="kafkaConfig">bulk</option>
+      <option name="basePath">bulk</option>
```

Comment:
`basePath` is only taken into account for a chronicle stream. This option should be defined in the chronicle part.